### PR TITLE
SASS-8133: Refactored field names for screens and audit events

### PR DESCRIPTION
**Pull request by vivekhmrc**

### CODE REVIEW

File: conf/application-json-logger.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="WARN"/>
+    <logger name="uk.gov" level="DEBUG"/>
```
Comment (arunkumar461) [inline conf/application-json-logger.xml:8]: lets not commit these changes? both xml files

**Comment by arunkumar461** [comment]
image

Unable to progress after this page

**Review by arunkumar461 — DISMISSED** [review]

File: conf/application-json-logger.xml (hunk 1 above)
Comment (vivekhmrc) [reply conf/application-json-logger.xml]: Configured for local testing. Removed.

**Review by modivilioglu — APPROVED** [review]
LGTM

**Pull request merged**